soh/src/overlays/actors/ovl_En_GirlA/z_en_girla.c

```diff
@@ -304,7 +307,10 @@ static ShopItemEntry shopItemEntries[] = {
       EnGirlA_ItemGive_BottledItem, EnGirlA_BuyEvent_ShieldDiscount },
     /* SI_RED_POTION_R50 */
     { OBJECT_GI_LIQUID, GID_POTION_RED, func_8002EBCC, 50, 1, 0x0065, 0x0063, GI_POTION_RED, EnGirlA_CanBuy_RedPotion,
-      EnGirlA_ItemGive_BottledItem, EnGirlA_BuyEvent_ShieldDiscount }
+      EnGirlA_ItemGive_BottledItem, EnGirlA_BuyEvent_ShieldDiscount },
+    /* SI_RANDOMIZED_ITEM */
```

Comment:
part of me wishes we weren't modifying vanilla tables like this (and vanilla methods like TryChangeShopItem, but I can't think of a better way to handle it and I'm not sure how to wrap that in a 64ddflag check

might be worth making a backlog issue for it (just a place to put ideas for other ways to handle this if we come up with any)

Reply:
Most of the progress on shopsanity was made before this was added, but adding this allowed me to vastly improve the readability and simplicity of the code. So I agree I don't love it, but I do prefer the code now to what it was before